layouts/register/types/not-needed.html

```diff
@@ -7,6 +7,11 @@ <h2>
 </header>
 <!--FAQPage Code Generated by https://saijogeorge.com/json-ld-schema-generator/faq/-->
 
+{{ $dates__electionday := $translation.register.dates__electionday }}
+{{ if and (eq .Params.state_name "Louisiana") (ne $translation.register.dates__electionday_la nil) }}
+{{ $dates__electionday = $translation.register.dates__electionday_la }}
+{{ end }}
```

Comment:
Once you have added the override to the state-data.json file you can recompile the content files and then you can use the same method as the deadline to simplify this logic into a single line.